yosh/builtins/export.py

```diff
@@ -0,0 +1,8 @@
+import os
+from yosh.constants import *
+
+
+def export(args):
+    var=args[0].split('=', 1 );
```

Comment:
[PEP8]
- missing whitespace around operator
- whitespace before ')'
- statement ends with a semicolon

Need to fix it to `var = args[0].split('=', 1)`